### DISCUSSION TITLE
Tagging and pruning

### DISCUSSION THREAD

**Discussion by rizlas** [bb-Ricardo/netbox-sync#96] [Q&A] [answered]
Hi,
I'm currentyl running your tool to sync three different vCenter sources. Can you explain how the tagging system works?
In particular how pruning is implemented.

For example, I run the tool with settings of vCenter X and I expect to find orphans tags only for items that belongs to vCenter X. Instead, in the logs I can read that items that belongs to vCenter Y and vCenter Z are being tagged as orphans. With pruning enabled all the items of Y and Z will be deleted.

At this point I assume that the tool is looking for every item that has "NetBox-synced" tag and if they can't be found in the vCenter specified in setting file it will mark them as Orphan. Is that right?

Thank you for your tool and time.

**Comment by bb-Ricardo** [comment, marked as answer]
Hi,

thank you for using it.

I described the process here https://github.com/bb-Ricardo/netbox-sync#pruning

If the item is not found in ***ANY*** configured source it is tagged as orphaned. If the source is `enabled = False` then pruning is disabled for this source.

Does this help?

**Reply by rizlas** [reply]
Hi,
thank you for the answer, i was creating three different setting files and that's why many orphans were generated.

**Reply by bb-Ricardo** [reply]
Yeah, this would cause this behavior. One config file with all sources and done.
